modules/cnf-installing-the-operators.adoc

```diff
@@ -157,21 +157,17 @@ metadata:
 spec:
   config:
     ignition:
-      version: 2.2.0
+      version: 3.1.0
     storage:
       files:
-        - contents:
+        - path: /etc/modprobe.d/sctp-blacklist.conf
+          mode: 0644
+          contents:
             source: data:,
-            verification: {}
-          filesystem: root
-          mode: 420
-          path: /etc/modprobe.d/sctp-blacklist.conf
-        - contents:
+        - path: /etc/modules-load.d/sctp-load.conf
+          mode: 0644
+          contents:
```

Comment:
In `modules/nw-sctp-enabling.adoc` there is a similar section about configuring SCTP and the MC uses `overwrite: true` for the `.conf` files.

I think we should do the same here for consistency, as this also discusses enabling SCTP